### ISSUE TITLE
Consolidate Map Files

### ISSUE THREAD

**Issue by Piras314** [Goob-Station/Goob-Station#1151]
we dont need _Goobstation versions for wizden maps

**Comment by BombasterDS2** [comment]
Won't the wizden updates break silos and BSO/Nanorep spawners on maps?

**Comment by SX-7** [comment]
I agree there, upstream does their own thing. We also avoid some of the controversial changes they add. Also reduces maintainer workload on upstream merges by automatically "skipping" map related PRs, which usually need to be done manually due to map file format diffs being weird asf. This is by all means intentional.
